src/contracts/atlas/ExecutionEnvironment.sol

```diff
@@ -334,11 +333,8 @@ contract ExecutionEnvironment is Base {
     function donateToBundler(address surplusRecipient)
         external
         payable
-        validPhase(ExecutionPhase.SolverOperations)
+        onlyIfFirstDonationInCurrentPhase
```

Comment:
Thoughts on moving the donateToBundler func to ExecutionBase?

Reply:
Yeah that's a more sensible place to put it. Will do that.